pkg/util/constants.go

```diff
@@ -0,0 +1,10 @@
+package util
```

Comment:
No package named util.  Also this is basically api stuff so `pkg/api` might be OK place to start.